source/alerts.txt

```diff
@@ -76,11 +76,11 @@ several **Events**:
          for Static Hosting
      - A Custom URL is confirmed for static hosting 
    
-   * - :ref:`Deployment <application-deployment>` 
+   * - :ref:`Deployment <deploy>`
```

Comment:
is this ref deleted? 

if so, should we keep it and put it so there are no potential downstream breakages?

Reply:
There are no downstream references in other docs repos and I'd prefer to not keep dangling refs around